### PR TITLE
ZMI upgraded to Zope4+

### PR DESCRIPTION
**Pull request by drfho**
Hi @dataflake,
I revised the ZMI-forms according to the Zope4+ look. 
Because of some unusal elements (copyright line, legend)  I added a small css change to the zope-base.css

Best regards
f

sql1
sql2
sql3

### CODE REVIEW

File: Products/SQLAlchemyDA/version.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1.0.0
```
Comment (dataflake) [inline Products/SQLAlchemyDA/version.txt:1]: Is there a reason you added this file? Looks like an accident.

**Review by dataflake — CHANGES_REQUESTED** [review]

File: Products/SQLAlchemyDA/version.txt (hunk 1 above)
Comment (drfho) [reply Products/SQLAlchemyDA/version.txt:1]: I am sorry, there is a reason: the version-file seems to be expected by the code. If the version-file is not avaible, SQLAlchemyDA/da.py throws FileNotFoundError (with Zope5)
filennotfound

**Comment by dataflake** [comment]
I would rip out any place that wants to use that `getVersion` method and the method code itself alongside that `version.txt` file.

**Comment by drfho** [comment]
Hi @dataflake 
thank you for the hint. Hope I did not remove the function calls it too radically:
https://github.com/zopefoundation/Products.SQLAlchemyDA/pull/9/commits/ed254898c1c8db6b43fcb9f59922998c308b4d52

**Review by dataflake — APPROVED** [review]
I restored the copyright notice and just removed the `getVersion` call

**Pull request merged**